festim/boundary_conditions/fluxes/surface_kinetics.py

```diff
@@ -0,0 +1,168 @@
+from festim import FluxBC
+from fenics import *
+import sympy as sp
+
+
+class SurfaceKinetics(FluxBC):
+    """
+    FluxBC subclass allowing to include surface processes in 1D H transport simulations
+
+    .. math::
+
+        \dfrac{d c_{\mathrm{s}}}{dt} = k_{\mathrm{bs}} c_{\mathrm{m}} \lambda_{\mathrm{abs}} (1 - \dfrac{c_\mathrm{s}}{n_{\mathrm{surf}}})
+        - k_{\mathrm{sb}} c_{\mathrm{s}} (1 - \dfrac{c_{\mathrm{m}}}{n_\mathrm{IS}}) + J_{\mathrm{vs}};
+
+        -D \dfrac{\partial c_{\mathrm{m}}}{\partial x} = -\lambda_{\mathrm{IS}} \dfrac{\partial c_{\mathrm{m}}}{\partial t}
+        - k_{\mathrm{bs}} c_{\mathrm{m}} \lambda_{\mathrm{abs}} (1 - \dfrac{c_\mathrm{s}}{n_{\mathrm{surf}}})
+        + k_{\mathrm{sb}} c_{\mathrm{s}} (1 - \dfrac{c_{\mathrm{m}}}{n_\mathrm{IS}}),
+
+    where :math:`c_{\mathrm{m}}` is the concentration of solute hydrogen, :math:`c_{\mathrm{s}}` is the surface concentration of hydrogen
```

Comment:
```suggestion
    where :math:`c_{\mathrm{m}}` is the concentration of mobile hydrogen (H/m3), :math:`c_{\mathrm{s}}` is the surface concentration of adsorbed hydrogen (H/m2)
```

Reply:
If math will work in API, the units can be shown as :math:`\mathrm{H m}^{-3}`, otherwise - OK

Reply:
`\mathrm{H \ m}^{-3}` then! $\mathrm{H \ m}^{-3}$